vms/organization/views.py

```diff
@@ -85,3 +87,34 @@ class OrganizationListView(LoginRequiredMixin, ListView):
     def get_queryset(self):
         organizations = Organization.objects.order_by('name')
         return organizations
+
+def approve(request, organization_id):
+    """ 
+    approves the pending organizatons
+    
+    :param organization_id: The id of the pending organization
+    :return: redirect to list of organizations 
+    """    
+    unlisted_org = get_organization_by_id(organization_id)
+    unlisted_org.approved_status = 1
+    unlisted_org.save()
+    return HttpResponseRedirect('/organization/list')
+
+def reject(request, organization_id):
+    """ 
+    rejects the pending organizatons
+    
+    :param organization_id: The id of the pending organization
+    :return: redirect to list of organizations 
+    """    
+    unlisted_org = get_organization_by_id(organization_id)
+    unlisted_org.approved_status = 2
+    unlisted_org.save()
+    vol_email = Volunteer.objects.get(organization=unlisted_org).email
+    try:
+        send_mail("Organization Rejected", "The organization you filled while sign-up has been rejected",
+             "messanger@localhost.com", [vol_email], fail_silently=False)
+    except:
+        raise Exception("There was an error in sending emails.")
```

Comment:
Instead of this can we redirect to 404 page or add in some UI changes, showing the message? Also we should keep a log of failures in my opinion. Same for other exceptions of this type in other PRs. Also be specific in error catching.